src/constants/fieldTypes.js

```diff
@@ -147,7 +147,7 @@ export const fieldTypeChoices = [
         descriptionId: 'DROPDOWN_CHOICE_HELPER_TEXT',
         value: 'choices',
         type: fieldTypes.optioneditor,
-        defaultValue: [],
+        defaultValue: null,
```

Comment:
Are you sure that this should change? Choices aren't an array any more?

Reply:
Since I couldn't detect any difference in behavior on the platform either way (I imagine that it makes a difference somewhere, but I can't easily find where), I've changed it back to an array here.